### ISSUE TITLE
 Embed code for dataset

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#127]
Users can embed the dataset on other sites or locations by using the "Embed" option. This option generates code automatically.

**Comment by philipashlock** [comment]
I think this issue was originally meant to focus on an embeddable view of the rows within a file, like the [recline.js](http://okfnlabs.org/recline/) previews we currently provide, but as suggested by @betson there's also a lot of value in providing an embeddable widget of the dataset metadata just as it's displayed on catalog.data.gov. This would provide consistency and reduce duplication for presenting the information on agency websites. 

![screen shot 2014-09-26 at 6 07 02 pm](https://cloud.githubusercontent.com/assets/183402/4428246/8af016a6-45c9-11e4-8e34-7740f009e549.png)

**Comment by rebeccawilliams** [comment]
+1 might be worth brainstorming a version that is desirable for data driven news outlets as well cc @rypan @paulschreiber

**Comment by paulschreiber** [comment]
What do you think, @dmil @ascheink?

**Comment by ascheink** [comment]
Yeah, we've talked about doing something like this for FiveThirtyEight. Definitely worth exploring.

**Comment by rebeccawilliams** [comment]
See also: the citation guidance being explored at [Project Open Data's Issue #458](https://github.com/project-open-data/project-open-data.github.io/issues/458). A citation format that included an embed would be great!
